### DISCUSSION TITLE
Why the file .prettierrc for the new projects?

### DISCUSSION THREAD

**Discussion by frederikhors** [sveltejs/kit#4232] [Ideas]
Can I ask you why the new app has that `.prettierrc`?

I think this is a choice of every single project which also depends on a business organization which is often managed by files like [`.editorconfig`](https://editorconfig.org/).

Can we remove it?

**Comment by hanneswidrig** [comment]
> Can I ask you why the new app has that `.prettierrc`?
> 
> 
> 
> I think this is a choice of every single project which also depends on a business organization which is often managed by files like [`.editorconfig`](https://editorconfig.org/).
> 
> 
> 
> Can we remove it?

I don't see the concern, the instant you create a new SvelteKit project, you can modify the Prettier config however you like.
